packages/e2e-tests/specs/editor/various/preview.test.js

```diff
@@ -272,7 +272,9 @@ describe( 'Preview', () => {
 		await publishPost();
 
 		// Close the panel.
-		await page.waitForSelector( '.editor-post-publish-panel' );
+		await page.waitForSelector(
+			'.editor-post-publish-panel__header button'
```

Comment:
@gwwar Isn't it generally better to try to match the label of the button? Maybe use XPath or match against the aria-label vs. just button within header class?

Reply:
yes, though I was reusing the existing selector below. @alexstine would you prefer I update both here?

Reply:
That was a suggestion in my own PR so I figured I'd mention it.

https://github.com/WordPress/gutenberg/pull/37934/files/02aa671ae11b58bf76f6dd991174937b1ba3ec1a#r808766732

I guess the upside is if the label changes, that would also fail the test adding another safeguard. Whatever you think is best. Change makes sense though, make sure button exists before clicking it.

Reply:
Sure, let's update it. I've added it in https://github.com/WordPress/gutenberg/pull/39100/commits/729d4d33b05f92c81ee7e58efd63ce493799cfab 

Thanks for taking a look!